src/services/control-plane/app/control-plane.ts

```diff
@@ -0,0 +1,18 @@
+import { ForManagerAuthentication } from "../ports/drivers";
+import { ForMonitoring } from '../ports/drivens/for-monitoring';
+import { AuthDetails,Permissions } from "./schemas";
+//Mi hexagono que tiene implementado los puertos de los driveR, en este caso es solo uno pero podrian ser mas
+export class ControlPlane implements ForManagerAuthentication{
+     //el Constructor los DriverN del puerto, osea que son los recurson necesario para que mi hexagono funcione
+     constructor(private readonly forMonitoring: ForMonitoring){}
+     
+     //Los metodos de mi hexagono que vienen de los driveR de los pruertos
+     async getAuthDetails(email: string, password: string): Promise<AuthDetails> {
+          
+          // const authDetails = await this.
+          return Promise.resolve({token:"123", refreshToken:"1234"})
```

Comment:
Aquí te faltan los adapters que te resuelven la authentication, normalmente utilizarías una tecnología como jwt o similares y para comunicarte con la misma necesitas de un adapter, así de esta manera la información que entra en la app y la que sale hacia afuera siguen útiles y sin conflictos

Reply:
Ahora mismo solo tienes los puertos de entrada al hexágono, que no esta mal, pero faltarían los puertos driven que se comunican con otros hexágonos o herramientas a nivel interno para obtener los recursos.
En tu caso te faltaría un for-authenticating al que luego le generarás un adapter que lo implemente de tipo stub y que este devuelva si está o no authenticado el user. 

La lógica que guardas en el servicio en sí, en tu caso el control plane, siempre es de negocio. Por ejemplo y fuera de este contexto, si la persona esta logueada generar un log, guardar en una base de datos aparte y devolver esta informacion al usuario es parte de la lógica de negocios; Pero cómo se realiza el log, cómo se guarda en la base de datos, es algo externo a la lógica de negocios, y es responsabilidad de otro servicio o herramienta aparte, en nuestro caso el logger es una herramienta y el manejo en la base de datos lo haría el servicio repository que ya tenemos hecho. Y para cada uno de estos casos deberíamos de tener un adapter que comunique la información